build/buildBuildImages.sh

```diff
@@ -533,11 +535,14 @@ elif [ "$imageTypeToBuild" == "vso-focal" ]; then
 	buildVsoImage "focal"
 elif [ "$imageTypeToBuild" == "vso-bullseye" ]; then
 	buildVsoImage "bullseye"
-elif [ "$imageTypeToBuild" == "cli" ]; then
+elif [ "$imageTypeToBuild" == "cli-stretch" ]; then
 	buildCliImage
 	buildCliImage "buster"
+	buildCliImage "bullseye"
```

Comment:
nit: should we only call `buildCliImage` if `cli-stretch` is provided?

Reply:
This comment still applies -- I think it makes sense to structure this chain of conditions as follows:

```bash
elif [ "$imageTypeToBuild" == "cli"]; then
    buildCliImage
    buildCliImage "buster"
    buildCliImage "bullseye"
elif [ "$imageTypeToBuild" == "cli-stretch"]; then
    buildCliImage
elif [ "$imageTypeToBuild" == "cli-buster"]; then
    buildCliImage "buster"
elif [ "$imageTypeToBuild" == "cli-bullseye"]; then
    buildCliImage "bullseye"
```

This gives us the ability to build all three flavors of CLI images with just the `cli` option, while also allowing us to build any of the individual flavors with a different option.

Reply:
Updated buildBuildImages.sh script to incorporate this change.